components/builder-worker/src/runner/mod.rs

```diff
@@ -418,7 +453,7 @@ impl Runner {
                         }
                         self.cancel();
                         self.cleanup();
-                        tx.send(self.job().clone()).map_err(Error::Mpsc)?;
+                        tx.send(self.job().clone()).await.unwrap(); // map_err(Error::Mpsc)?;
```

Comment:
Can we use `map_err(Error::Mpsc)?;` here?

Reply:
Addressed in a33367f0febdd930dfb5cba3892af203fc5b3a51